app/client/components/accountoverview/accountOverviewCard.tsx

```diff
@@ -51,13 +51,26 @@ export const AccountOverviewCard = (props: AccountOverviewCardProps) => {
 
   const subscriptionStartDate = props.productDetail.subscription.start;
 
+  const subscriptionEndDate = props.productDetail.subscription.end;
+
   const nextPaymentDetails = getNextPaymentDetails(
     mainPlan,
     props.productDetail.subscription,
     null,
     hasPaymentFailure
   );
 
+  // We want to hide manage sub/payment buttons and the trial period if the product is a gifted digi sub
+  const isDigiSubGift =
+    specificProductType.friendlyName === "digital subscription" &&
+    isGift(props.productDetail.subscription);
+
+  const userIsGifter = isDigiSubGift && props.productDetail.isPaidTier;
+
+  const giftPurchaseDate = props.productDetail.subscription.lastPaymentDate;
+
+  const shouldNotShowStartDate = shouldShowJoinDateNotStartDate || userIsGifter;
```

Comment:
As we negate this when we use it perhaps it should just be `shouldShowStartDate`